interfaces/security_test.go

```diff
@@ -120,11 +153,9 @@ func (s *SecuritySuite) TestSecCompPlugPermissions(c *C) {
 	// Ensure that plug-side security profile looks correct.
 	blobs, err := s.repo.SecurityFilesForSnap(s.plug.Snap)
 	c.Assert(err, IsNil)
-	c.Check(blobs, DeepEquals, map[string][]byte{
-		"/run/snappy/security/seccomp/producer/hook.profile": []byte("" +
-			"# TODO: add default seccomp profile here\n" +
-			"allow open\n"),
-	})
+	c.Check(blobs["/var/lib/snappy/seccomp/profiles/producer.origin_hook_version"], DeepEquals, []byte(""+
+		"# Mocked seccomp header\n"+
+		"allow open\n"))
```

Comment:
Note, this isn't a valid seccomp profile entry. It would be nicer to have:
"# Mocked seccomp header\n"
"open\n"))